plugins/asdf/asdf.plugin.sh

```diff
@@ -0,0 +1,26 @@
+#! bash oh-my-bash.module
+# asdf.plugin.sh: asdf plugin for oh-my-bash.
+
+# Custom ASDF_DIR location
+if [[ -v ASDF_DIR ]]; then
+  . "$ASDF_DIR/asdf.sh"
+
+# Standard install location
+elif [[ -f "${XDG_CONFIG_HOME:-$HOME}/.asdf/asdf.sh" ]]; then
+  ASDF_DIR="${XDG_CONFIG_HOME:-$HOME}/.asdf"
+  . "${ASDF_DIR}/asdf.sh"
+
+# Arch Linux / AUR Package
+elif [[ -f "/opt/asdf-vm/asdf.sh" ]]; then
+  ASDF_DIR="/opt/asdf-vm"
+  . /opt/asdf-vm/asdf.sh
+
+# Homebrew
+elif _omb_util_command_exists brew; then
+  __ASDF_PREFIX="$(brew --prefix asdf)"
```

Comment:
Is this an internal variable of this plugin? Can we change the variable name as follows?

```suggestion
  _omb_plugin_asdf__prefix="$(brew --prefix asdf)"
```

If so, the other occurrences of `__ASDF_PREFIX` should also be updated.

Reply:
> If so, the other occurrences of `__ASDF_PREFIX` should also be updated.

Reply:
Good catch, thanks. Fixed up.

Reply:
Thanks.